docs/source/tutorials/outliers.ipynb

```diff
@@ -534,9 +534,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3 (ipykernel)",
+   "display_name": "cleanlab-p38",
    "language": "python",
-   "name": "python3"
+   "name": "cleanlab-p38"
```

Comment:
IMPORTANT: for the docs to build properly in CI, notebooks must have kernelspec that looks like this:
```
"kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
...
```

instead of referencing arbitrary virtualenvs

Reply:
I've fixed this